commec/tools/search_handler.py

```diff
@@ -55,8 +79,21 @@ def temp_log_file(self):
         """Temporary log file used for this search. Based on outfile name."""
         return f"{self.out_file}.log.tmp"
 
-    @abstractmethod
     def search(self):
+        """
+        Wrapper for _search, to ensure that it is only called if 
+         - The output doesn't already exist,
+         - If force is enabled.
+        """
+        if not self.force and self.check_output():
+            logging.info("%s expected output data already exists, "
+                         "will use existing data found in:\n%s",
+                         self.__class__.__name__, self.out_file)
+            return
```

Comment:
I think I support splitting this into `--resume` and `--force` (as discussed on our call)